vendor/src/github.com/hyperhq/libcompose/docker/container.go

```diff
@@ -317,20 +317,15 @@ func (c *Container) Run(ctx context.Context, imageName string, configOverride *c
 	})
 
 	if err := c.client.ContainerStart(ctx, container.ID, ""); err != nil {
-		return -1, err
+		return -1, container.ID, err
 	}
 
 	if err := <-errCh; err != nil {
 		logrus.Debugf("Error hijack: %s", err)
-		return -1, err
+		return -1, container.ID, err
 	}
 
```

Comment:
Why not do containerWait here? Then we do not need to change the interface, right?